### PR TITLE
Add class files

### PR DESCRIPTION
**Pull request by isc-ddo**

### CODE REVIEW

File: data/src/cls/Pet.cls [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/// PURPOSE: This class stores information about pets. To populate 100 sample pets, you can run: do ##class(Demo.Pet).Populate(100)
+Class Demo.Pet Extends (%Persistent, %Populate)
+{
+
+Property Name As %String(POPSPEC = "##class(%PopulateUtils).FirstName()");
+
+Property Type As %String(VALUELIST = ",dog,bird,cat,rat,mouse,rabbit");
```
Comment (isc-zkrowiak) [inline data/src/cls/Pet.cls:7]: just double checking that the leading period is necesssary in the valuelist?

File: data/src/cls/Pet.cls [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Property Color As %String;
+
+Property PetShop As Demo.PetShop;
+
+Index ColorIdx On Color [ Type = bitmap ];
```
Comment (isc-zkrowiak) [inline data/src/cls/Pet.cls:13]: is this the correct way to name indices?  Could these be "ColorIndex"? Or are these changes out of scope?
```suggestion
Index ColorIndex On Color [ Type = bitmap ];
```

**Review by isc-zkrowiak — APPROVED** [review]
A few questions, but everything looks good.

File: data/src/cls/Pet.cls (hunk 2 above)
Comment (isc-ddo) [reply data/src/cls/Pet.cls:13]: Thank you @zkrowiak. Your suggested changes make sense to me. I will talk to Jenny about it.

**Pull request merged**